<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowCentres</class>
 <widget class="QDialog" name="WindowCentres">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>539</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Gestion Centres</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#QWidget {

background:#404040;

}

QTabWidget::pane { /* The tab widget frame */

border-top: 5px solid #535353;

}

QTabWidget::tab-bar {

left: 0px; /* move to the right by 5px */

}

/* Style the tab using the tab sub-control. Note that it reads QTabBar _not_ QTabWidget */

QTabBar::tab {

background: #dcdcdc;

border: 2px solid #535353;

border-bottom-color: rgb(0, 0, 0); /* same as the pane color */

border-top-left-radius: 7px;

border-top-right-radius: 7px;

min-width: 120px;

padding: 8px;

color: rgb(0, 0, 184);

font-size:15px;

font-family:Arial, Helvetica, sans-serif;

}

QTabBar::tab:selected, QTabBar::tab:hover {

background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #c3a1a1, stop: 0.4 rgb(85, 0, 255), stop: 0.5 rgb(67, 0, 202), stop: 1.0 #3f00bd);
color :white;
}

QTabBar::tab:selected {

border-color: rgb(0, 0, 0);

border-bottom-color: #535353; /* same as pane color */

color: #dcdcdc;

font-size:15px;

font-family:Arial, Helvetica, sans-serif;

}

QTabBar::tab:!selected {

margin-top: 2px; /* make non-selected tabs look smaller */

}

</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="ajouter">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>8</number>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>ajouter</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="6" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>nombre des recruteurs</string>
         </property>
        </widget>
       </item>
       <item row="8" column="6" colspan="2">
        <widget class="QPushButton" name="c_pb_ajouter">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>ajouter</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>tel</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>email</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>nom</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="c_telephone">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::PreventContextMenu</enum>
         </property>
         <property name="accessibleName">
          <string/>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="c_nom">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_21">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>ville</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>adresse</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="ville">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="c_email">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="c_nomrecruteurs">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="c_adresse">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="8" column="8">
        <spacer name="horizontalSpacer_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget">
      <attribute name="title">
       <string>supprimer</string>
      </attribute>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>90</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>nom</string>
       </property>
      </widget>
      <widget class="QPushButton" name="c_pushButton_2">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>160</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QComboBox" name="c_comboBox">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>80</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>afficher</string>
      </attribute>
      <widget class="QTableView" name="tab_centre">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>631</width>
         <height>192</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
      <widget class="QPushButton" name="c_pushButton">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>240</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Afficher</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>modifier</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="7" column="2">
        <spacer name="horizontalSpacer_3">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>159</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="3">
        <spacer name="horizontalSpacer_4">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>160</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>tel</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="c_telupdated">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="c_nbupdated">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="c_comboBox_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="ville_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>nom</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>email</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QPushButton" name="c_pushButton_4">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>modifier</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_22">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>ville</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>adresse</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="c_adresseupdated">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>nombre des recruteurs</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="c_emailupdated">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>mailing</string>
      </attribute>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>A:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>100</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Objet:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="c_ob">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>100</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>140</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>description:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="p_envoyer_5">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>80</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Envoyer</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="c_des">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>140</y>
         <width>251</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
      <widget class="QComboBox" name="c_mailing">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>localisation</string>
      </attribute>
      <widget class="QQuickWidget" name="quickWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>611</width>
         <height>261</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="resizeMode">
        <enum>QQuickWidget::SizeRootObjectToView</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>10</y>
         <width>311</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>             CENTRE DE RECRUTEMENT PROFESSIONNEL CRP</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>statistique</string>
      </attribute>
      <widget class="QTableView" name="tab_r">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>291</width>
         <height>211</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
      <widget class="QPushButton" name="c_pushButton_3">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>130</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Afficher</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>271</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>CENTRES ET NOMBRE DES RECRUTEURS</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>gérer les réclamations </string>
      </attribute>
      <widget class="QPushButton" name="c_pb_reclamation_5">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>200</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>gérer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>100</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>recherche</string>
      </attribute>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nom du centre</string>
       </property>
      </widget>
      <widget class="QComboBox" name="c_comboBox_3">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
      <widget class="QPushButton" name="c_pb_chercher">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>50</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>chercher</string>
       </property>
      </widget>
      <widget class="QTableView" name="tab_rech">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>311</width>
         <height>171</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>50</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ville</string>
       </property>
      </widget>
      <widget class="QComboBox" name="ville_3">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>50</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
      <widget class="QTableView" name="tab_ville">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>81</y>
         <width>311</width>
         <height>171</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
      <widget class="QPushButton" name="c_chercher2">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>50</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>chercher</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
